### ISSUE TITLE
Load the log for pre-train

### ISSUE THREAD

**Issue by hvt1609** [pfnet/pytorch-pfn-extras#59]
How to load the log and continue draw the loss png when I pre-train the model ?

**Comment by emcastillo** [comment]
This should be done automatically if you are using the snapshot extensions.

**Comment by hvt1609** [comment]
let try it,

Thank.
